lib/logstash/outputs/kafka.rb

```diff
@@ -140,6 +140,8 @@ class LogStash::Outputs::Kafka < LogStash::Outputs::Base
   config :ssl_keystore_password, :validate => :password
   # The password of the private key in the key store file.
   config :ssl_key_password, :validate => :password
+  # Algorithm to use when verifying host. Set to "" to disable
+  config :ssl_endpoint_identification_algorithm, :validate => :string
```

Comment:
Can you set the value explicitly to 'https' to ensure that the behavior remains the same in the (probably somewhat unlikely) event that the Kafka client changes defaults again